packages/message-encryption/README.md

```diff
@@ -0,0 +1,65 @@
+# `@waku/message-encryption`
+
+Provide Waku Message Version 1 payload encryption as defined in [26/WAKU2-PAYLOAD](https://rfc.vac.dev/spec/26/).
+
+## Symmetric Encryption
+
+Symmetric encryption uses a unique key to encrypt and decrypt messages.
+
```

Comment:
can we add info on when is it recommended to use which type of encryption/decryption?

Reply:
> can we add info on when is it recommended to use which type of encryption/decryption for the end user who might not know/care much about it

Well it's in the sentence, it uses a _unique key_ for both encryption and decryption.

I'd prefer to have more verbose explanations in https://waku.guide/ especially around recommendations as we don't want to have to update a recommendation in 5 different places.